iati/core/tests/test_resources.py

```diff
@@ -1,6 +1,7 @@
 """A module containing tests for the library implementation of accessing resources."""
 from lxml import etree
 import pytest
+from six import string_types
```

Comment:
`six` keeps on causing problems in other code. Is there a way to implement this functionality without having to use it?